conductor_api/src/conductor/admin.rs

```diff
@@ -1182,28 +1197,52 @@ type = 'http'"#,
     }
 
     #[test]
+    #[cfg(not(windows))]
```

Comment:
@willemolding please use `#[cfg(feature = "broken-tests")]` rather than elide for windows

otherwise there is no way of knowing what is permanently/by design not applicable to windows and what is a poor test

Reply:
Is there a way to do that but also have it run for the other non-windows OS?

Reply:
Yeah, I would like to keep the test in for our Linux tests on CircleCI. We definitely have to log this with an issue for future investigation and we could also just link this PR in a comment on this line. But I definitely prefer `#[cfg(not(windows))]` over `#[cfg(feature = "broken-tests")]` in this case.

Reply:
@thedavidmeister made the change you suggested in Mattermost `#[cfg(any(not(windows), feature = "broken-tests"))]`. CI is passing looks like we are all good